packages/@aws-cdk/aws-apigatewayv2-integrations/lib/websocket/lambda.ts

```diff
@@ -36,9 +36,16 @@ export class LambdaWebSocketIntegration implements IWebSocketRouteIntegration {
       }),
     });
 
+    const integrationUri = Stack.of(route).formatArn({
```

Comment:
https://github.com/aws/aws-cdk/blob/aa9fd9cd9bbaea4149927e08d57d29e547933f49/packages/%40aws-cdk/aws-apigatewayv2-integrations/lib/http/lambda.ts#L51

Looks like this should also be fixed.

Reply:
I could not find anywhere in the [docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html) for HTTP API (or CFN) about this. I also deployed the integration stack and could see the console working fine for HTTP APIs. 

In the CFN [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integration.html#cfn-apigatewayv2-integration-integrationuri) it is mentioned that this should be URI of lambda function (not sure if that means ARN?).

I think we should leave this as is since it might cause unnecessary regression. What do you think?